server/tables/tables.go

```diff
@@ -670,6 +670,34 @@ func (*QuotaGroup) TableName() string {
 	return "QuotaGroups"
 }
 
+type EncryptionKey struct {
+	Model
+	KeyID   string `gorm:"primaryKey"`
```

Comment:
wdyt about calling it EncryptionKeyID to be consistent with the `${TableName}ID` pk-naming pattern? e.g. `APIKeyID`